src/lib/utils/typography/Typography.styles.tsx

```diff
@@ -0,0 +1,96 @@
+import { makeStyles } from "@material-ui/core";
+
+const basicFontStyle = {
+  fontFamily: "Lato",
+  fontStyle: "normal",
+  letterSpacing: "0px",
+}
+
+const useTypographyStyle = makeStyles(
```

Comment:
**Not needed for this PR**, but would like to have a demo in StyleGuidist with all of these variants and a link to the Figma. Any other ways we can close the gap between design tokens and the end result?

Reply:
That's a good idea.